### ISSUE TITLE
Publish x3dom-full on npm

### ISSUE THREAD

**Issue by medihack** [x3dom/x3dom#732]
I am trying to use x3dom for volume rendering in a Webpack environment (Vue.js). So I installed [x3dom from npm](https://www.npmjs.com/package/x3dom). Unfortunately, it doesn't seem to include the volume rendering component (rendering some surface meshes work fine). Would it be possible to publish `x3dom-full` on npm?

**Comment by andreasplesch** [comment]
Perhaps best to contact the maintainer directly:

https://www.npmjs.com/~fibo

**Comment by medihack** [comment]
Isn't it an official repository of x3dom? Wouldn't it make sense to publish it automatically with every new release? It seems straight forward as I only added a `package.json` and it [seems to work perfectly](https://github.com/medihack/x3dom-full-dist). More and more people are using module bundlers (like Webpack or Rollup), so it would make the adoption much easier for new users.

**Comment by andreasplesch** [comment]
No, I do not believe the npm package is maintained by the x3dom developers. They (FHG) probably want to avoid any overhead on top of maintaining the github repo and the existing system.
Although I cannot speak for the x3dom team, I would suspect that adding automatic publishing to npm (perhaps first to an external account) to the (python) build system would be probably a welcome contribution as long as it does not add any manual work load.

**Comment by fibo** [comment]
@medihack I asked maintainers to package x3dom and they decided to keep it in another repo: https://github.com/x3dom/dist

I also asked if it was worth to add files x3dom-full.js and x3dom-full-physics.js and at that time they said it was ok.

Can you please tell me which files would you add to the package? I tryed to see your link https://github.com/medihack/x3dom-full-dist but now it gives 404.

Question: do you think it could be feasible to import x3dom-full.js like this?

```javascript
import x3dom from 'x3dom/full'
```

If yes, I could add it in the full/ folder as index.js without breaking backward compatibility for other users.

**Comment by jamesleesaunders** [comment]
Can we get version 1.8 published to npm?

**Comment by fibo** [comment]
@jamesleesaunders just published v1.8
